datacite/__init__.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of DataCite.
+#
+# Copyright (C) 2012, 2013, 2014, 2015 CERN.
+#
+# DataCite is free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+
+"""Python API wrapper for the DataCite Metadata Store API.
+
+API documentation is available on https://mds.datacite.org/static/apidoc.
+"""
+
+from __future__ import absolute_import, unicode_literals, print_function
+
+from ._compat import b
+from .errors import DataCiteError
+from .request import DataCiteRequest
+
+
+class DataCiteMDSClient(object):
```

Comment:
What about moving this implementation into say `client.py`, so that `__init__.py` can stay ultra short file, with no embedded business logic, only exporting symbols we want to export?